### DISCUSSION TITLE
Specify number of comments to display when using the issue explorer

### DISCUSSION THREAD

**Discussion by ScaryOrca** [ankitpokhrel/jira-cli#511] [Ideas] [answered]
When using the list option to view issues, there doesn't appear to be a way to control the number of comments displayed like there is when using the view option. I primarily use the interactive UI, so being able to view comments without having to open another terminal would be incredibly useful.

**Comment by timka** [comment]
AFAIU, `v` calls `jira issue view` under the hood. So if there's a way to set the defaults for  the `issue view` command (`--comments` equivalent)  in config file this should be enough to achieve the desired effect.

**Comment by justageek** [comment]
I second this request, would be a great feature.

**Comment by ankitpokhrel** [comment, marked as answer]
This is done with #853 and will be available in the next release. You can either use `--comments` flag with `issue list` cmd or set `num_comments` key to desired number in your config.
